src/legacy/core_plugins/metric_vis/public/metric_vis_fn.js

```diff
@@ -32,21 +32,114 @@ export const metric = () => ({
     defaultMessage: 'Metric visualization'
   }),
   args: {
-    visConfig: {
-      types: ['string', 'null'],
-      default: '"{}"',
+    percentage: {
+      types: ['boolean'],
+      default: false,
+      help: i18n.translate('metricVis.function.percentage.help', {
+        defaultMessage: 'Shows metric in percentage mode. Dont forget to set colorRange.'
+      })
+    },
+    colorSchema: {
+      types: ['string'],
+      default: '"Green to Red"',
+      help: i18n.translate('metricVis.function.colorSchema.help', {
+        defaultMessage: 'Color schema to use'
```

Comment:
Is there any way to describe this besides "Color schema"? It feels like the type of thing that wouldn't be immediately intuitive to a user

Reply:
Also (I don't know the answer to this): should colorSchema have a list of options? e.g.
```javascript
options: ['Green to Red', 'Blah to blahblah'],
```

Reply:
"Color scheme" (not schema) seems like the right name here too